keras/models.py

```diff
@@ -100,28 +112,74 @@ def compile(self, optimizer, loss, class_mode="categorical", theano_mode=None):
             test_ins = [self.X_test, self.y]
             predict_ins = [self.X_test]
 
-        self._train = theano.function(train_ins, train_loss, 
+        self._train = theano.function(train_ins, train_loss,
             updates=updates, allow_input_downcast=True, mode=theano_mode)
-        self._train_with_acc = theano.function(train_ins, [train_loss, train_accuracy], 
+        self._train_with_acc = theano.function(train_ins, [train_loss, train_accuracy],
             updates=updates, allow_input_downcast=True, mode=theano_mode)
-        self._predict = theano.function(predict_ins, self.y_test, 
+        self._predict = theano.function(predict_ins, self.y_test,
             allow_input_downcast=True, mode=theano_mode)
-        self._test = theano.function(test_ins, test_score, 
+        self._test = theano.function(test_ins, test_score,
             allow_input_downcast=True, mode=theano_mode)
-        self._test_with_acc = theano.function(test_ins, [test_score, test_accuracy], 
+        self._test_with_acc = theano.function(test_ins, [test_score, test_accuracy],
             allow_input_downcast=True, mode=theano_mode)
 
 
+class Sequential(Model):
```

Comment:
In modern Keras, the Sequential model is build through double inheritance from the Sequential container and the generic Model. Why not here?